src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,23 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        User userOne = new User("emailOne", "passwordOne");
+        userService.add(userOne);
+        System.out.println(userService.findByEmail(userOne.getLogin()));
+        AuthenticationService authenticationService
+                = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        authenticationService.register("emailTwo", "passwordTwo");
+        authenticationService.register("emailNew", "passwordNew");
+        authenticationService.login("emailOne", "passwordOne");
```

Comment:
```suggestion
        authenticationService.register("emailTwo", "passwordTwo").sout;
        authenticationService.register("emailNew", "passwordNew").sout;
        authenticationService.register("emailNew", "passwordNew").sout;
        authenticationService.login("emailOne", "passwordOne").sout;
        authenticationService.login("emailOne", "wrongpassword").sout;
        authenticationService.login("wrongemail", "passwordOne").sout;
```
for proper testing )

Reply:
I don`t understand why we need to add try-catch in main if we have  already try catch in proper Dao an Controllers class?

Reply:
and with sout we only see the user entity...  I think the same almost as just like default query messages of the hibernate ...